libs/viewer/src/Settings.cpp

```diff
@@ -0,0 +1,834 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by mIcable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <viewer/Settings.h>
+
+#include <utils/Log.h>
+
+#include <sstream>
+#include <string>
+
+#include <assert.h>
+
+// NOTE: We are including cgltf only to get JSMN. We should consider adding jsmn to third_party.
```

Comment:
Is this comment still needed / can we include the jsmn headers directly now?

Reply:
I removed this comment in a subsequent CL.

Reply:
whoops, missed that